paper/paper.md

```diff
@@ -0,0 +1,127 @@
+infrastructure. An equitable, or just, transition further requires the
+recognition of the people and communities directly affected by this transition.
+However, public preferences may be ignored during decision-making processes
+related energy infrastructure due to a lack of technical rigor or expertise
+[@johnson:2021]. This challenge is further complicated by the fact that people
+have and express preferences over many dimensions simultaneously.
+Multi-objective optimization offers a method to help decision makers and
+stakeholders understand the problem and analyze tradeoffs among solutions
+[@liebman:1976]. Although, to date, no multi-objective energy modelling
+frameworks exist. Open-source multi-objective energy system framework (`osier`)
+is a Python package for designing and optimizing energy systems across an
+arbitrary number of dimensions. `osier` was designed to help localized
+communities articulate their energy preferences in a technical manner without
+requiring extensive technical expertise. In order to facilitate more robust
+tradeoff analysis, `osier` generates a set of  technology portfolios, called a
+Pareto front, with multi-objective optimization using evolutionary algorithms.
+`osier` also extends the common
+modelling-to-generate-alternatives (MGA) algorithm into many dimensions,
+allowing users to investigate the near-optimal for appealing alternative
+solutions. In this way, `osier` may aid modelers in addressing procedural and
+recognition justice.
+
+# Statement of Need
+There are myriad open- and closed-source energy system optimization models
+(ESOMs) available [@pfenninger:2022]. ESOMs can be used for a variety of tasks
+but are most frequently used for prescriptive analyses meant to guide
+decision-makers in planning processes. However, despite the many available
+models, all of these tools share a fundamental characteristic: Optimization over
+a single economic objective (e.g., total cost or social welfare).
+Simultaneously, there is growing awareness of energy justice and calls for its
+inclusion in energy models [@pfenninger:2014; @vagero:2023]. Some studies
+attempted to incorporate local preferences into energy system design through
+multi-criteria decision analysis (MCDA) and community focus groups
+[@bertsch:2016; @mckenna:2018; @zelt:2019]. But these studies rely on tools with
+pre-defined objectives which are difficult to modify. Without the ability to add
+objectives that reflect the concerns of a community, the priorities of that
+community will remain secondary to those of modellers and decision
+makers. A flexible and extensible multi-objective framework that fulfills this
+need has not yet been developed. The `osier` framework closes this gap.
+
+# Design and Implementation
+In order to run `osier`, users are only required to supply an energy demand time
+series. Users can optionally provide weather data to incorporate solar or wind
```

Comment:
Is this the only thing they are required to provide? What about what technologies they want to use and the limitations of the model?